src/Controller/PythagoreController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace App\Controller;
+
+use App\Utils\PythagoreUtility;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class PythagoreController extends AbstractController
+{
+    private $pythagoreUtility;
```

Comment:
```suggestion
    private PythagoreUtility $pythagoreUtility;
```
on type les variables